package.json

```diff
@@ -59,12 +59,13 @@
     "view-cover": "istanbul report html && opn ./coverage/index.html",
     "browser": "run-browser test/index.js",
     "phantom": "run-browser test/index.js -b | tap-spec",
-    "dist": "browserify --standalone virtual-dom index.js > dist/virtual-dom.js",
+    "dist": "mkdir -p dist; browserify --standalone virtual-dom index.js > dist/virtual-dom.js",
```

Comment:
Thanks for the link! I once looked for binary node packages to replace Unix programs like *cp*, *mv* or *mkdir* – but gave up seeing that no-one really bothers to make them.

I [can’t find](http://npmsearch.com/?q=mkdir) a binary alternative to *mkdir* – so I’ll push a `node --eval` solution.